app/models/claim.rb

```diff
@@ -131,7 +131,7 @@ def can_access_youth_court_flow?
   def can_claim_youth_court?
     nsm? &&
       can_access_youth_court_flow? &&
-      plea_category.match?(/category_[12]a/)
+      plea_category.match?(/category_(?:2|[12]a)$/)
```

Comment:
at this point, ['category_1a', 'category_2a'].include?(plea_category) feels more readable - but as I've said before that's more of a personal take because I hate regex so won't let that hold off the merge